bucket/nvui.json

```diff
@@ -0,0 +1,33 @@
+{
+    "version": "0.2.1",
+    "description": "Fast and feature-rich front-end for neovim",
+    "homepage": "https://github.com/rohit-px2/nvui",
+    "license": "MIT",
+    "suggest": {
+        "neovim": "neovim"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/rohit-px2/nvui/releases/download/v0.2.1/nvui-win64.zip",
+            "hash": "5c455f6705d76b4a7afcb6af31de627caad81514f166ce7c0e3f33de0d31703f"
+        }
+    },
+    "extract_dir": "nvui",
+    "pre_install": "if (!(Test-Path \"$persist_dir\\bin\\nvui-config.ini\")) {New-Item \"$dir\\bin\\nvui-config.ini\" -ItemType File | Out-Null}",
+    "bin": "bin\\nvui.exe",
+    "shortcuts": [
+        [
+            "bin\\nvui.exe",
+            "nvui"
+        ]
+    ],
+    "persist": "bin\\nvui-config.ini",
```

Comment:
Any docs for this file? What does it store?

(IIRC all config is done through neovim itself).

Reply:
I don't have docs for this file, but this commit says `UI & Window settings`.
 https://github.com/rohit-px2/nvui/commit/a9f170251531e6cc9252426dd19596c3a9ce8fad